roles/openshift_sanitize_inventory/tasks/__deprecations_logging.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Well, agreed; but the goal is to have this info flash up almost instantly, not half an hour into the install. If you wait until the logging role is running to check its variables, that's quite a way into the install. This is why I've been advocating for running roles in multiple phases, so we can encapsulate role-specific validations within the role, but run them at an appropriate time in the install. Until we have that, this seems expedient.

A longer wait would make the notice more likely to be seen, and provide a little more incentive to actually make the change. Say 30s?

@michaelgugino How would you suggest indicating deprecation of variables? About the only other way that makes sense to me would be a callback plugin that notes them at the end of the run (and I'm not sure how likely that is to be seen either depending on how many other callback plugins operate). Almost the only messaging available in Ansible is to fail a task; there doesn't seem to be a way to provide a nuanced warning that users will actually see.